helpers/helpers.tf

```diff
@@ -0,0 +1,60 @@
+// IMPORTANT!!!
```

Comment:
Rather than using a symbolic link, let's take an idiomatic approach and make this a private submodule which outputs the locals. Submodules are considered private by convention if they lack a README.

Reply:
Great idea! Will update to use a submodule.